README.md

```diff
@@ -247,34 +242,27 @@ browser on iOS) are interesting cases in which the underlying browser engine on
 doesn't match the engine that the relevant browser built themselves. What should we do in these
 cases?
 
-I think we have a few options for the string:
+We have a few options for the string:
```

Comment:
Maybe "There are a few..."? Not sure if it really matters all that much.

Reply:
Sure